test/e2e/reconciler/ingress/controller_test.go

```diff
@@ -204,10 +189,14 @@ func TestIngressController(t *testing.T) {
 
 			t.Log("Installing sink cluster...")
 			start := time.Now()
-			_, err = framework.CreateClusterAndWait(t, ctx, source.Artifact, sourceKcpClusterClient.Cluster(clusterName), sink)
+			workloadCluster, err := framework.CreateReadyCluster(
+				t, ctx, source.Artifact, sourceKcpClusterClient.Cluster(clusterName), sink)
 			require.NoError(t, err)
 			t.Logf("Installed sink cluster after %s", time.Since(start))
 
+			resources := sets.NewString("ingresses.networking.k8s.io", "deployments.apps", "services")
+			framework.StartWorkspaceSyncer(t, resources, workloadCluster, source, sink)
```

Comment:
WorkspaceSyncer or WorkloadSyncer?

Reply:
It's 'workspace syncer' because it's targeting a workspace instead of a pcluster. Also, this code is likely to change when we get kind involved (soon).

Reply:
ack